DESCRIPTION

```diff
@@ -0,0 +1,24 @@
+Package: regexTestR
+Title: R Regex Tester
+Version: 0.0.0.9000
+Authors@R: c(
+    person('Adam', 'Spannbauer', email = 'SpanbauerAdam@gmail.com', role = c('cre', 'aut')),
+    person('Chong', 'Kim', email = 'chong.kim@ucdenver.edu', role = c('ctb'))
+    )
+Description: Shiny application that provides R users an application to test/check regular expression query in R.
+License: What license is it under?
+Encoding: UTF-8
+LazyData: true
+Imports: 
+    shiny,
+    golem,
+    data.table,
+    shinyBS,
+    stringr,
+    purrr,
+    tidyr,
+    htmltools,
+    processx
```

Comment:
Some initial feedback.  Given that #10 appears to be the main motivation for `golem`-ifying the app, I think we should be exhaustive about the dependencies of the app here.  The below is one of the warnings from running `devtools::check()`.

> ```
>❯ checking dependencies in R code ... WARNING
>  '::' or ':::' imports not declared from:
>    ‘DT’ ‘RColorBrewer’ ‘rvest’ ‘shinythemes’ ‘xml2’
> ```

There are other issues brought up by running this check, but as of now, I'm not planning on deploying the package to CRAN.  If you'd like to get the package to a CRAN-ready state, then we would need to get all Errors, Warnings, & Notes down to 0.

Reply:
I certainly haven't thought about getting the app CRAN-ready but maybe I could push that forward. I've added/updated the dependencies.